README.md

```diff
@@ -1 +1,93 @@
-# bookstore2.0
+# bookstore
+Bookstore is an app that let's its users to add new book, view list of books and remove a book. Built with React, Redux, and Tailwind CSS.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# bookstore
+
+> Bookstore is an app that let's its users to add new book, view list of books and remove a book. Built with React, Redux, and Tailwind CSS.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- React
+- Stylelint, Web Hint,
+- Eslint, Prettier
+
+## Live Demo
+
+[Live Demo Link]()
+
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of HTML, CSS, JavaScript, and React
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/topeogunleye/bookstore/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone https://github.com/topeogunleye/bookstore.git
+```
```

Comment:
- [ ] Let's try to be thorough in our documentation, as it can be of great help to users. Your link for setting up the project locally is wrong. Kindly fix this. Here is the correct link: **`https://github.com/topeogunleye/bookstore2.0.git`**